static/src/stylesheets/module/content-garnett/_media.head.scss

```diff
@@ -52,3 +52,19 @@
         background: $brightness-20;
     }
 }
+
+.content--media.content--pillar-news {
```

Comment:
Is it the only pillar affected? (others are `opinion`, `lifestyle`, `arts`, and `sport`)